Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -59,6 +59,7 @@ def decompile(self, data, ttFont):
 		noname = 0
 		self.glyphs = {}
 		self.glyphOrder = glyphOrder = ttFont.getGlyphOrder()
+		next = 0
 		for i in range(0, len(loca)-1):
```

Comment:
Maybe you can rename the local variable nextOffset to avoid shadowing the built-in next() function. I know it is not used in this scope but syntax highlighters aren't that smart and the blue coloured label is sometimes confusing/distracting.

Reply:
Agreed. I renamed `last` to `pos`, so `next` became `nextPos`.